### ISSUE TITLE
Password protected Vimeo links don't work

### ISSUE THREAD

**Issue by DavidKipling** [sampotts/plyr#1879]
When using a Vimeo link which is password protected I see the password box but I can't enter a value

![image](https://user-images.githubusercontent.com/67172014/85183704-8b41e480-b284-11ea-8813-cac3b818f3a5.png)

### Expected behaviour
Allow entry of password

### Actual behaviour
Unable to enter password

### Steps to reproduce
Use vimeo video with a password as the source

**Comment by CodeDruid13** [comment]
They won't. That's why they are password protected. Am using private Vimeo videos and i have to use an Api to get the video links with tokens to use

**Comment by DavidKipling** [comment]
@CodeDruid13 Thanks for your reply, much appreciated.  Sorry I'm new to using Vimeo links, are you saying I can use vimeo links with passwords if I access the Vimeo API first to get the video link?  Would this work if I put my own password box up first to get the password and then call the API? or is the API access limited to the owner of the video?

My scenario is users of the site add videos so I'm not in control or have access to the Vimeo account.

**Comment by CodeDruid13** [comment]
> @CodeDruid13 Thanks for your reply, much appreciated. Sorry I'm new to using Vimeo links, are you saying I can use vimeo links with passwords if I access the Vimeo API first to get the video link? Would this work if I put my own password box up first to get the password and then call the API? or is the API access limited to the owner of the video?
> 
> My scenario is users of the site add videos so I'm not in control or have access to the Vimeo account.

Yeah. You'll be acces the password protected videos only if you're the owner or you've been added as a user to the associated Vimeo account

**Comment by CodeDruid13** [comment]
> @CodeDruid13 Thanks for your reply, much appreciated. Sorry I'm new to using Vimeo links, are you saying I can use vimeo links with passwords if I access the Vimeo API first to get the video link? Would this work if I put my own password box up first to get the password and then call the API? or is the API access limited to the owner of the video?
> 
> My scenario is users of the site add videos so I'm not in control or have access to the Vimeo account.

Unfortunately for protected videos, the user will have to enter password over and over anytime they need to view the video. Vimeo restricts access for password protected videos.

**Comment by Lapin** [comment]
Issue seems to be related to the wrapper/container, not the password protection system of Vimeo. After all plyr.js uses official Vimeo API to grab the videos. @DavidKipling mentions the text input for the password is not targetable hence he cannot input any password. 

@DavidKipling: not sure it'd work but on the demo site there's a plyr option `referrerPolicy: 'no-referrer'` with a note saying `Prevent Vimeo blocking plyr.io demo site`. Worth to try that first. If it doesn't, I'd suggest to so a crude embed test on jsfiddle or codepen first and then see if it's working. Just the video -don't forget to add plyr.css as well. If it's working there, then it should be something overlaying on the frontend. Both vimeo and youtube API's has pretty funky sizing options that would break your front end easily.

**Comment by sanuas** [comment]
**_Add this CSS_**

```
.plyr--vimeo .plyr__video-embed .plyr__poster {
      z-index: -1 !important;
}
```

**Comment by ltroya-as** [comment]
Is there a way to know if the video is password protected? That way we can add our own implementation
